modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/SeriesEntity.java

```diff
@@ -79,20 +79,21 @@
   @Column(name = "access_control", length = 65535)
   protected String accessControl;
 
+  @Lob
   @ElementCollection(targetClass = String.class)
-  @MapKeyColumn(name = "name")
-  @Column(name = "value")
+  @MapKeyColumn(name = "name", nullable = false)
+  @Column(name = "value", length = 65535)
   @CollectionTable(name = "oc_series_property", joinColumns = {
-          @JoinColumn(name = "series", referencedColumnName = "id"),
-          @JoinColumn(name = "organization", referencedColumnName = "organization") })
+          @JoinColumn(name = "series", referencedColumnName = "id", nullable = false),
```

Comment:
No primary key `(organization, series, name)` is generated for this table.

Reply:
Same here.